reviewrot/__init__.py

```diff
@@ -1,20 +1,32 @@
 import collections
 import logging
 import os
+import datetime
+import re
+import argparse
 from os.path import expanduser, expandvars
 from shutil import copyfile
 from six.moves import input
 from six import iteritems
-
+from dateutil.relativedelta import relativedelta
 from reviewrot.gerritstack import GerritService
 from reviewrot.githubstack import GithubService
 from reviewrot.gitlabstack import GitlabService
 from reviewrot.pagurestack import PagureService
+from reviewrot.basereview import Age
 
 import yaml
 
 log = logging.getLogger(__name__)
 
+# Valid values of choices for arguments
+FORMAT_CHOICES = ['oneline', 'indented', 'json']
```

Comment:
I would do FORMAT_CHOICES and SORT_CHOICES or CHOICES = {'format': [...], 'sort': [...]}, but not both.

Reply:
fixed